### ISSUE TITLE
Error With Custom Filtering: FooTable.components.core is undefined (v3)

### ISSUE THREAD

**Issue by jeffconklin** [fooplugins/FooTable#518]
Hello,

I recently updated to 3.1.0 and found that any table where I extended your filtering (like what is shared on http://fooplugins.github.io/FooTable/docs/examples/advanced/filter-dropdown.html) now errors with:
TypeError: FooTable.components.core is undefined

At this line within my code:
FooTable.components.core.register('filtering', FooTable.MyStatusFiltering_Combined);

After reviewing step 6 on the link above, I see it should not reference core anymore. Once I updated it to the following, it worked properly:
FooTable.components.register('filtering', FooTable.MyStatusFiltering_Combined);

Just wanted to add this issue/solution here in case anyone else has the same issue.

Also, I would like to ask that if you do put out releases that change / deprecate previous implementations, that there is a note about it within the changelog and on any example pages.

Keep up the good work.

**Comment by steveush** [comment]
Hi Jeff,

It's all ready in the changelog, has it's own special note in the readme on the main page https://github.com/fooplugins/FooTable, and all the examples were updated.

Thanks

**Comment by jeffconklin** [comment]
You are 100% right, but I did not read that note until now. I jumped to the 2 logical places.. within the specific changelog section (specifically there and not the other note), as well as the example page (http://fooplugins.github.io/FooTable/docs/examples/advanced/filter-dropdown.html). Now that you pointed out the special note above the change log I see it. But including a "code has changed see note elsewhere" in the changelog, as well as a specific change note on the example page would help.

Thanks again.
